docker/index.yaml

```diff
@@ -26,7 +26,7 @@ artifactory.algol60.net/csm-docker/stable:
 
     # cray-sat is not included in any Helm charts
     cray-sat:
-      - 3.33.1
+      - 3.33.*
```

Comment:
Temporary change to test glob processing. Represents a sample change which we may have in integration branch. Will be reverted before merge into release branch.